app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,75 @@
+class TweetsController < ApplicationController
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect to "/tweets/new"
+    else
+    @tweet = Tweet.new(content: params[:content])
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.new(content: params[:content], user: current_user)
```